src/vsr/replica.zig

```diff
@@ -6759,8 +6816,8 @@ pub fn ReplicaType(
                         self.view_headers.array.get(0).op,
                     },
                 );
-                self.repair_messages_budget -|= 1;
-                if (self.repair_messages_budget > 0) {
+                if (self.repair_messages_budget_journal > 0) {
+                    self.repair_messages_budget_journal -|= 1;
```

Comment:
Here and below, just:

```suggestion
                    self.repair_messages_budget_journal -= 1;
```

given that we are inside the if anyway.